stable/grafana/templates/configmap-dashboard-provider.yaml

```diff
@@ -16,11 +16,11 @@ data:
   provider.yaml: |-
     apiVersion: 1
     providers:
-    - name: 'default'
-      orgId: 1
-      folder: ''
-      type: file
-      disableDeletion: false
+    - name: '{{ .Values.sidecar.dashboards.provider.name }}'
```

Comment:
can you please document these in the values.yaml?

Reply:
They already are included in the values.yaml and documented there. Did you miss them or are they not sufficient, in which case I gladly extend them.

Reply:
i'm sorry, i think one and wrote another :(. I meat to document in the readme.md